### ISSUE TITLE
Chargeback/Refund Suggestion.

### ISSUE THREAD

**Issue by Consue** [GingerWalnut/SQ5.0Public#284]
The core idea behind this system is to remove the dependence of trust on player to player extanges.

The feature tries to equate the rights of the buyer and seller and is similar to PayPal refund system but more automatic.

Basically when someone sends money to a recipient, the giver has the option to enable a time-sensitive refund. Let’s say 5k was given to a pirate as a protection fee, if the service is inadequate to the buyer (ie the pirate starts attacking the buyer) the player can instantly refund the transaction. As such the system is less dependent trusting that the seller will carry out thier end of the deal.

This would remove a lot of unnecessary toxic and dispute when it comes to transaction ie. Resource trade, protection agreements, blockade fees and alliances trusts and so on.

I mentioned before that the refund is time sensitive, ie after a certain amount of time the buyer can no longer refund. This is to minimize the time between the service/good being delivered and the refund expiration.

As with anything, this system can be exploited, especially with material transactions such as resource/ ship trade. This is because after the seller has already transferred the items, the buyer can basically call a refund and therefore end up with items they did not pay for. As such for large transactions (100k and up) a mod should be notified to be a broker if disputes do arise. Also the ability to refund can be removed the instant target items enter the buyers inventory or there is a ship owner transfer.

In summary I think this is a step in the right direction for achieving a egalitarian, trustless monitary system but it needs to be strictly moderated to prevent loopholes.
